<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>831</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="planes_bn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>planes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="company_bn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>company</string>
    </property>
   </widget>
   <widget class="QPushButton" name="status_bn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hangar_bn">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>hangar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shedule_bn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>shedule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="way_bn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>way</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_bn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="commit_bn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>commit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_bn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_bn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>831</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_bn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_bn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeDBFolder_bn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>change DB</string>
    </property>
   </widget>
   <widget class="QLabel" name="DirPath">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>430</y>
      <width>821</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>path</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
